### PR TITLE
Integrates origami-lib with TravisCI

### PR DESCRIPTION
**Pull request by idealadarsh**
Adds a .travis.yml file and runs some script to install the library.
No tests are currently there so it just installs origami-lib for now.

My fork:
https://github.com/idealadarsh/origami-lib

Travis build:
https://travis-ci.org/idealadarsh/origami-lib/builds/311200222

Related GCI task: 
https://codein.withgoogle.com/dashboard/task-instances/4717290908024832/

### CODE REVIEW

**Comment by AvaisP** [comment]
#6 added tests. Please make sure Travis is running the tests

**Comment by idealadarsh** [comment]
Thanks! Will do that.

**Comment by idealadarsh** [comment]
@AvaisP Tests run now.

**Comment by idealadarsh** [comment]
@AviasP Did you check the builds?
They are failing because tests are running.
Tests are throwing errors, I don't know who wrote those tests and how to fix them.

But this issue is solved.
Travis is integrated and tests are running.

**Comment by idealadarsh** [comment]
@AvaisP Sorry I didn't see that. Please wait a minute

**Comment by idealadarsh** [comment]
@AvaisP Please check this build, server is running but tests are not.
https://travis-ci.org/Cloud-CV/origami-lib/builds/311312466

**Comment by idealadarsh** [comment]
@AvaisP It is giving same error when I ran those tests locally as in travis.
Here's the log: https://pastebin.com/ScCVvyh6

**Comment by AvaisP** [comment]
Check the comment on the task

**Comment by AvaisP** [comment]
Great job! Suggesting a few changes

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+sudo: required
+language: python
+
+python:
+  - "2.7"
```
Comment (AvaisP) [inline .travis.yml]: Does it not run with python 3.4+?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+addons:
+  apt:
+    packages:
+      - python-dev
+      - python-numpy
```
Comment (AvaisP) [inline .travis.yml]: Why are you installing numpy and opencv  by apt and then by pip?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  - pip install opencv-python
+  - pip install -r requirements.txt
+  - python origami-lib/test/server_textimageinput.py &
+  - sleep 1
+
+script:
+  - cd origami-lib/test/
+  - pytest
+
+cache: pip
```
Comment (AvaisP) [inline .travis.yml]: Always end your file with a new line

**Comment by idealadarsh** [comment]
@AvaisP No it fails with python3.4, check this build: https://travis-ci.org/idealadarsh/origami-lib/builds/311379467

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  - pip install pytest
+  - pip install numpy
+  - pip install opencv-python
+  - pip install -r requirements.txt
+  - python origami-lib/test/server_textimageinput.py &
+  - sleep 1
+
+script:
+  - cd origami-lib/test/
+  - pytest
```
Comment (AvaisP) [inline .travis.yml]: Please shut the server after script

Comment (idealadarsh) [reply .travis.yml]: How?

Comment (AvaisP) [reply .travis.yml]: Nevermind. I'll make that a different task

File: .travis.yml (hunk 1 above)
Comment (AvaisP) [reply .travis.yml]: Will create a task for this

Comment (idealadarsh) [reply .travis.yml]: Ok

File: .travis.yml (hunk 4 above)
Comment (idealadarsh) [reply .travis.yml]: Good for me for now

**Pull request merged**